app/views/products/_product_form.html.erb

```diff
@@ -0,0 +1,109 @@
+<%= form_for [@vendor, @product], html: { class: "form-horizontal" } do |f| %>
+
+  <div class = "form-group">
+    <%= f.label :name, "Product Name", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-4">
+      <%= f.text_field :name, :class => "form-control", :placeholder => "Name", :required => true %>
+    </div>
+    <div class = "col-sm-2" >* required</div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :version, "Version", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-4">
+      <%= f.text_field :version, :class => "form-control", :placeholder => "Version"%>
+    </div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :description, "Description", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-4">
+      <%= f.text_area :description, :class => "form-control", :placeholder => "Write product description here..."%>
+    </div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :ehr_type, "EHR Type", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-2">
```

Comment:
Both `col-sm-2` can be consolidated to a `col-sm-4` and then you can wrap the two buttons in `<div class = "btn-group btn-group-justified" role="group"></div>` and have the desired effect without extra CSS